src/dune_rules/pkg_rules.ml

```diff
@@ -195,6 +186,48 @@ module Install_cookie = struct
     | Some f -> f
 end
 
+module Env_update = struct
+  include Dune_lang.Action.Env_update
+
+  let update kind ~new_v ~old_v ~f =
+    if new_v = "" then old_v
```

Comment:
Why is setting the new value to an empty string a no-op?

Reply:
Looks like I misunderstood the spec. Will fix.

Reply:
Actually, this is the correct behavior. From the manual:

```
FOO += "", FOO := "", etc. are all ignored - i.e. opam never adds empty segments to an existing variable.
```